nginx-directory/html.libsonnet

```diff
@@ -0,0 +1,18 @@
+local k = import 'k.libsonnet';
```

Comment:
Late to this one, a question: Can we use `ksonnet-util/kausal.libsonnet` and `k.libsonnet` interchangeably?

I noticed that `nginx-directory/configfile.libsonnet` uses `ksonnet-util/kausal.libsonnet`.

Reply:
think of kausal.libsonnet as k.libsonnet + utils. If you are just using raw k8s, just use k.libsonnet.

Reply:
Although, there are some opinions that we might loose there - e.g. default labels, etc, which is worth thinking about.